library/core/src/macros/mod.rs

```diff
@@ -816,6 +816,7 @@ pub(crate) mod builtin {
     #[macro_export]
     macro_rules! env {
         ($name:expr $(,)?) => {{ /* compiler built-in */ }};
+        ($name:expr, $error_msg:expr) => {{ /* compiler built-in */ }};
```

Comment:
Trailing commas are allowed for this form too. https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=dab96e07abece7b8906c773b82dc588e
```suggestion
        ($name:expr, $error_msg:expr $(,)?) => {{ /* compiler built-in */ }};
```